admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -150,15 +146,21 @@ main() {
         effect: NoSchedule'
     fi
 
-    TLS_PRIVATE_KEY=$(cat "$SERVER_KEY" | base64 -w 0) \
+    if [ -x "$(command -v envsubst)" ]; then
+        ENVSUBST="$(command -v envsubst) < $TEMPLATE_PATH"
+    else
```

Comment:
Maybe `elif [ -x "$(command -v docker)" ]`, and fail if docker isn't around either?